docker/resources/build_fuzzy_libmath_dockerfile.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+DOCKERFILE=Dockerfile.mcalibmath
+
+generate_docker() {
+    
+cat > ${DOCKERFILE} <<HERE
+# Base image
+FROM ${1}
+
+# Copy fuzzy environment from fuzzy image
+RUN mkdir -p /opt/mca-libmath
+COPY --from=verificarlo/fuzzy:v0.4.1-lapack /opt/mca-libmath/libmath.so /opt/mca-libmath/
+COPY --from=verificarlo/fuzzy:v0.4.1-lapack /usr/local/lib/libinterflop* /usr/local/lib/
+
+# Set environment variables
+ENV VFC_BACKENDS 'libinterflop_mca.so --precision-binary32=24 --precision-binary64=53 --mode=mca'
+ENV LD_PRELOAD '/opt/mca-libmath/libmath.so'
+HERE
+    
+}
+
+build_docker() {
+    docker build -f ${DOCKERFILE} -t ${1} .
+}
+
+if [[ $# < 2 ]]; then
+    echo "usage: ./build_fuzzy_libmath_dockerfile <DOCKER_IMAGE> <TAG>"
```

Comment:
There could be a line of explanation for each parameter.

Reply:
Added